Sources/TerminalAPIKit/Models/SessionContainer.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  SessionContainer.swift
+//  TerminalAPIKit
+//
+//  Created by Andrew Gates on 8/8/23.
+//
+// Used in AdminRequests
+// https://docs.adyen.com/point-of-sale/shopper-engagement/create-session/
+
+import Foundation
+
+public final class SessionContainer: Codable {
+    public let session:Session
+    public let operation:[Operation]?
```

Comment:
looks like, according to the [docs](https://docs.adyen.com/point-of-sale/shopper-engagement/create-session/#start-session), the `operation` is mandatory, so the type of it should probably be non-optional